### PR TITLE
versioning of the node running script

### PR DESCRIPTION
**Pull request by nicolasburtey**
#### Test Coverage Data

* Before
* After

### CODE REVIEW

**Comment by nicolasburtey** [comment]
the command to access it post-merge should be:

wget https://raw.github.com/harmony-one/harmony/master/scripts/node.sh

File: scripts/node.sh [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+fi
+
+NODE_PORT=9000
+BC_MA=/ip4/54.183.5.66/tcp/9999/ipfs/QmdQVypu6NSm7m8bNZj5EJCnjPhXR8QyRmDnDBidxGaHWi
+
+# Kill existing soldier/node
+fuser -k -n tcp $NODE_PORT
+
+# Run Harmony Node
+nohup ./harmony -bc_addr $BC_MA -ip $PUB_IP -port $NODE_PORT > harmony-${PUB_IP}.log 2>&1
```
Comment (LeoHChen) [inline scripts/node.sh]: please add an '&' at the end of this line. nohup won't put the program in the background by default.

Comment (nicolasburtey) [reply scripts/node.sh]: should be good now

Comment (nicolasburtey) [reply scripts/node.sh]: for some reasons jenkis failed (I would doubt this is related to this commit). can you confirm approve so that I can merge?

**Pull request merged**